crates/re_types/src/components/keypoint_id.rs

```diff
@@ -0,0 +1,21 @@
+// NOTE: This file was autogenerated by re_types_builder; DO NOT EDIT.
+
+/// A 16-bit ID representing a type of semantic keypoint within a class.
+///
+/// `KeypointId`s are only meaningful within the context of a `crate::components::ClassDescription`.
+///
+/// Used to look up an `crate::components::AnnotationInfo` for a Keypoint within the `crate::components::AnnotationContext`.
```

Comment:
nit: doc link missing. Another thing where I'm wondering if codegen could help us a lil bit to do the right thing

Reply:
it's missing on purpose because I havent ported these objects to HOPE yet! and i didn't want to link back to legacy objects until we actually start backporting.